src/nn.h

```diff
@@ -342,6 +342,18 @@ NN_EXPORT  struct nn_cmsghdr *nn_cmsg_nxthdr_ (
 /*  Send/recv options.                                                        */
 #define NN_DONTWAIT 1
 
+/*  Global options.                                                           */
+#define NN_FORK_STRATEGY 1
+
+/*  Fork strategies.                                                          */
+/*  ABI warning: do not reorder, add new entries before NN_FORK_MAX_.         */
+enum nn_fork_strategy_kind {
+    NN_FORK_NONE = 1,
+    NN_FORK_RESET,
+
+    NN_FORK_MAX_
+};
+
```

Comment:
Going from memory here, but this might be the first use of an `enum` instead of a simple `int` in the nanomsg library here. (There exist merits for both design decisions, and it's worth considering the design decisions and precedents)